packages/fxa-auth-server/backstage.yaml

```diff
@@ -0,0 +1,42 @@
+---
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: fxa-auth-server
+  description: Auth Server back-end for Mozilla Accounts
+  tags:
+    - typescript
+    - javascript
+    - node
+    - hapi
+  annotations:
+    sentry.io/project-slug: mozilla/fxa-auth-server
+    circleci.com/project-slug: github/mozilla/fxa
+spec:
+  type: service
+  lifecycle: production
+  owner: fxa-team
+  system: mozilla-accounts
+  providesApis:
+    - api:fxa-auth
+  dependsOn:
+    - fxa-browserid-verifier
```

Comment:
Depends on profile api?

Reply:
Hmm, is this a direct requirement, or indirect via SQS? I don't recall us directly hitting it from auth-server.

Reply:
There's a profile client in the auth server, that makes rest calls via axios to profile server. I haven't dug any further than that. It could be dead code...